lua/caramba/git.lua

```
@@ -496,16 +496,21 @@ M.review_code = function()
   local utils = require("caramba.utils")
   local context = require("caramba.context")
   local llm = require("caramba.llm")
+  local intelligence = require("caramba.intelligence")
   
```

Luacheck (1.2.0)

[warning] line contains only whitespace

(W611)